Source/Krypton Toolkit/Examples/ComboBoxExtApp/ComboBoxExtApp.csproj

```diff
@@ -8,7 +8,7 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <ProjectReference Include="..\..\Main\Krypton.Toolkit.Suite.Extended.ComboBoxExt\Krypton.Toolkit.Suite.Extended.ComboBoxExt.csproj" />
+        <ProjectReference Include="..\..\Main\Krypton.Toolkit.Suite.Extended.ComboBoxExt\Krypton.Toolkit.Suite.Extended.Combo.Box.csproj" />
```

Comment:
I do not like these changes>
DataGridView is a single named entity, and the namespace should reflect that.

Reply:
Ok, will change to all one word. I don't think we need the `Ext`, as the extensions to them are already implied in the namespace.